### PR TITLE
change logs for next release

### PR DESCRIPTION
**Pull request by jennyf19**
I tried to get everything, especially customer facing stuff, but we've had a lot of changes.

### CODE REVIEW

File: adal/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+   - Fix NullPointException when adalv3 account is used because there is no client info in adalv3 [MSAL Issue #653](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
+   - RemoveAdalUser api now removes users by accountID and by displayableID to cover migration scenario from adalv3 to adalv4 to msalv2. [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
+   - Fix for internal TokenCache.Clear api so ADAL persistent cache is also cleared [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
+   - Optimize cache lookups with native iOS by modifying field for persistence on iOS. [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)
+   - Fix for reading ExtraQueryParameter environment variable on netcore in ADAL [ADAL issue #1362](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1362)
+- Make a ConfidentialClient entry point obsolete on platforms which only support confidential clients [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1375)
```
Comment (henrik-me) [inline adal/changelog.txt]: >- [](start = 0, length = 1)

nit: indentation

Comment (henrik-me) [inline adal/changelog.txt]: >-  [](start = 0, length = 2)

Cache fixes and this one should have the same indentation method, imo (consider matching with the MSAL changelog, that means a missing hyphen for Cache fixes) #Resolved

File: msal/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+   - RemoveAdalUser api now removes users by accountID and by displayableID to cover migration scenario from adalv3 to adalv4 to msalv2. [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
+   - Fix for internal TokenCache.Clear api so ADAL persistent cache is also cleared [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
+   - Optimize cache lookups with native iOS by modifying field for persistence on iOS. [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)
+- Support [extended lifetime tokens](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1331)
+- Remove Chrome dependency for system browser on Android devices. See https://aka.ms/msal-net-system-browsers for more information. [MSAL issue #664](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/664)
+- Add support for b2clogin.com for b2c authorities [MSAL issue #669](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/669) [#632](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/632)
```
Comment (henrik-me) [inline msal/changelog.txt:16]: >Add support for b2clogin.com for b2c authoritie [](start = 2, length = 47)

:) #Resolved

File: msal/changelog.txt [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+   - Fix NullPointException when adalv3 account is used because there is no client info in adalv3 [MSAL Issue #653](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
+   - RemoveAdalUser api now removes users by accountID and by displayableID to cover migration scenario from adalv3 to adalv4 to msalv2. [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
+   - Fix for internal TokenCache.Clear api so ADAL persistent cache is also cleared [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
+   - Optimize cache lookups with native iOS by modifying field for persistence on iOS. [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)
+- Support [extended lifetime tokens](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1331)
+- Remove Chrome dependency for system browser on Android devices. See https://aka.ms/msal-net-system-browsers for more information. [MSAL issue #664](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/664)
```
Comment (henrik-me) [inline msal/changelog.txt]: >Remove Chrome dependency for system browser [](start = 2, length = 43)

:) #Resolved

**Review by henrik-me — APPROVED** [review]
:shipit:

File: adal/changelog.txt (hunk 1 above)
Comment (jennyf19) [reply adal/changelog.txt]: sounds good.

---
In reply to: [231366029](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231366029) [](ancestors = 231366029)

File: msal/changelog.txt (hunk 3 above)
Comment (jennyf19) [reply msal/changelog.txt]: :face_with_rolling_eyes:

---
In reply to: [231366239](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231366239) [](ancestors = 231366239)

File: msal/changelog.txt (hunk 2 above)
Comment (jennyf19) [reply msal/changelog.txt:16]: :face_with_rolling_eyes:

---
In reply to: [231366199](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231366199) [](ancestors = 231366199)

File: adal/changelog.txt (hunk 1 above)
Comment (jmprieur) [reply adal/changelog.txt]: I would propose:
Version 4.4.0
===============
Improvements and fixes to the token cache
- The serialized token cache can now be shared by different applications, therefore providing SSO if the same user signs-in in both applications
  - See [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1365) and [MSAL Issue #653](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
- On .NET Core, the Token cache was shared by all instances of applications in memory. This is now fixed (See MSAL.NET issue #656 and [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1360))
- Fixes consistency issues for advanced token cache migration scenarios from ADAL v3.x to ADA v4.x to MSAL v2.x 
  - [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
  - [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
- Cache lookups were optimized. Work done in conjunction with ADAL.iOS and MSAL.iOS native) [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)
- The ExtraQueryParameter environment variable is now read on ADAL for the .NET Core platform, as was already the case in the .NET Framework case [ADAL issue #1362](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1362)
- A Client credential API which should not be used on Xamarin.iOS, Xamarin.Android and UWP is now marked as obsolete 
  - [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1375)
 #Resolved

File: msal/changelog.txt (hunk 3 above)
Comment (jmprieur) [reply msal/changelog.txt]: I would propose:
2.4.0-preview
=============
Improvements and fixes to the token cache
- The serialized token cache can now be shared by different applications, therefore providing SSO if the same user signs-in in both applications
  - See [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1365) and [MSAL Issue #653](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
- On .NET Core, the Token cache was shared by all instances of applications in memory. This is now fixed (See MSAL.NET issue #656 and [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1360))
- Fixes consistency issues for advanced token cache migration scenarios from ADAL v3.x to ADA v4.x to MSAL v2.x 
  - [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
  - [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
- Cache lookups were optimized. Work done in conjunction with ADAL.iOS and MSAL.iOS native) [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)

More browsers are now supported on Xamarin.Android when you choose to use system web browsers.
- Removed chrome dependency for system browser on Android devices. See https://aka.ms/msal-net-system-browsers for more information. [MSAL issue #664](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/664)

Azure AD B2C improvement
- Add support for b2clogin.com for b2c authorities [MSAL issue #669](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/669) [#632](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/632)
 #Resolved

**Review by jmprieur — APPROVED** [review]
LGTM, but proposed improvements :)

File: msal/changelog.txt (hunk 3 above)
Comment (jennyf19) [reply msal/changelog.txt]: this is great...thank you. we needed your PM magic. I  did a copy/paste

---
In reply to: [231409353](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231409353) [](ancestors = 231409353)

File: adal/changelog.txt (hunk 1 above)
Comment (jennyf19) [reply adal/changelog.txt]: perfect...thank you.

---
In reply to: [231409302](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231409302) [](ancestors = 231409302)

**Pull request merged**